AnnoyancesFilter/Popups/sections/antiadblock.txt

```diff
@@ -2204,6 +2206,9 @@ goduke.com,uwaterloo.ca##div[aria-label="Ad Blocker Detected"]
 ! SECTION[id=general]: Anti-adblock - Common '!#safari_cb_affinity(general)' directive
 !#safari_cb_affinity(general)
 !
+@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$domain=talksport.com
```

Comment:
Better with content type - it is just a HEAD request for adblock check, the script does not load

<img width="1882" height="308" alt="Image" src="https://github.com/user-attachments/assets/4b287d1a-00fc-44b9-8de5-c798ed901851" />

```suggestion
@@||pagead2.googlesyndication.com/pagead/js/adsbygoogle.js$xmlhttprequest,domain=talksport.com
```